doc/modules/clustering.rst

```diff
@@ -799,6 +805,88 @@ by black points below.
    In Proceedings of the 2nd International Conference on Knowledge Discovery
    and Data Mining, Portland, OR, AAAI Press, pp. 226–231. 1996
 
+.. _optics:
+
+OPTICS
+======
+
+The :class:`OPTICS` algorithm shares many similarities with the :class:`DBSCAN`
+algorithm, and can in fact be considered a generalization of DBSCAN that
+relaxes the ``eps`` requirement from a single value to a value range. The key
+difference between DBSCAN and OPTICS is that the OPTICS algorithm builds a
+*reachability* graph, which assigns each sample both a ``_reachability``
```

Comment:
if _reachability is private it's weird to document it in narrative doc.

Reply:
it's public-- just a typo as "_reachability" instead of "reachability_"  ...fixed